tests/test_canopy_cover.py

```diff
@@ -152,7 +181,7 @@ def test_good_input():
     os.makedirs(out_dir)
 
     try:
-        cmd = f'{SOURCE_PATH} --working_space {out_dir} --metadata {META} {INPUT1}'
+        cmd = f'{SOURCE_PATH} {INPUT1} --working_space {out_dir} --metadata {META}'
```

Comment:
Is this change still needed

Reply:
When it is not included then it says that there is no image file to work with
![Screen Shot 2021-01-15 at 9 36 10 AM](https://user-images.githubusercontent.com/62153719/104753371-1b5de480-5715-11eb-9219-02ce0f0f2e49.png)

Reply:
I added a test for no file_list/no image files

Reply:
I changed it back; I didn't scroll to the end of the message so I didn't see the {INPUT1}